### DISCUSSION TITLE
Missing Authentication Header in Proxy

### DISCUSSION THREAD

**Discussion by lebr0nli** [encode/httpx#1959] [Potential Issue]
I follow [httpx doc](https://www.python-httpx.org/advanced/#authentication)
```python
>>> httpx.get('http://google.com/', proxies={'http://':'http://aaaa:bbbb@127.0.0.1:8030'})
```

Although:
```python
>>> httpx.Proxy('http://aaaa:bbbb@127.0.0.1:8030')
Proxy(url='http://127.0.0.1:8030', headers={'proxy-authorization': 'Basic YWFhYTpiYmJi'})
```

I got:
```
GET http://google.com/ HTTP/1.1
Host: google.com
Accept: */*
Accept-Encoding: gzip, deflate, br
Connection: keep-alive
User-Agent: python-httpx/0.21.1
```

There is no `Proxy-Authorization`.
Did I miss using something?
Thanks!

**Comment by Dantenerosas** [comment]
I've got the same problem now actually. My proxies require now manually specifying proxy-authorization header unlike 0.20.0 version so I assume something is broken. Maybe it's something a bit related to #1960 , because it may be because something was removed from HTTPProxy\Transport objects. Seems like for now solution is to add it manually

**Reply by lovelydinosaur** [reply]
Thanks folks - opened as #1970

**Reply by lovelydinosaur** [reply]
Now resolved in the master version of `httpcore` - next release will close this off.

**Reply by lovelydinosaur** [reply]
Done
